nx-X11/programs/Xserver/hw/nxagent/Keystroke.c

```diff
@@ -249,6 +249,7 @@ static void parse_keystroke_file(void)
   char *homefile = "/.nx/config/keystrokes.cfg";
   char *etcfile = "/etc/nxagent/keystrokes.cfg";
 
+  /* FIXME: we do not want to have x2go references int the code */
```

Comment:
OK, I thought we wanted to get rid of that completely. Dropping that comment for now.